StudentsDB/src/main/java/com/musala/simple/students/db/DataBase.java

```diff
@@ -0,0 +1,19 @@
+package com.musala.simple.students.db;
+
+public interface DataBase {
+	
+	public void insert(Student student);
+	
+	public Student getStudentById(int id);
+	
+	public void deleteDb();
```

Comment:
I don't think there should be a method for dropping the db. In any real-life scenario, this is to be done manually by the DB admin ;) So get rid of this.